### PR TITLE
i18n(zh-cn): update `why-astro.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- update `why-astro.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 1]
```diff
@@ -61,7 +63,7 @@ Astro 的魔力在于它如何将上述两个值（内容焦点于服务器优
 
 我们首先确保您可以使用您已经知道的任何喜欢的 UI 组件语言。在 Astro 中创建新的 UI 组件时都支持React、Preact、Svelte、Vue、Solid、Lit 和其他一些组件。
 
-我们为了 Astro 也能有一个很好的内置组件语言，我们创建了自己 `.astro` UI语言。它很大程度上深受 HTML 的影响：任何有效的 HTML 部分都已经是有效的 Astro组件，它还结合了我们从其他组件中借用的一些功能，如：React的JSX表达式和 CSS范围（默认为Svelte和Vue）。
+我们为了 Astro 也能有一个很好的内置组件语言，我们创建了自己 `.astro` UI语言。它很大程度上深受 HTML 的影响：任何有效的 HTML 部分都已经是有效的 Astro组件，它还结合了我们从其他组件中借用的一些功能，如：React的JSX表达式和（像Svelte和Vue一样的）默认CSS 作用域。这种与 HTML 的相似性也使得使用渐进式增强和通用可访问性模式变得更加容易，而无需任何开销。
```
Comment (zihan-ch) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:66]:
```suggestion
我们为了 Astro 也能有一个很好的内置组件语言，我们创建了自己 `.astro` UI语言。它很大程度上深受 HTML 的影响：任何有效的 HTML 部分都已经是有效的 Astro 组件，它还结合了我们从其他组件中借用的一些功能，如：React 的 JSX 表达式和（像 Svelte 和 Vue 一样的）默认 CSS 作用域。这种与 HTML 的相似性也使得使用渐进式增强和通用可访问性模式变得更加容易，而无需任何开销。
```

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 2]
```diff
@@ -74,6 +76,6 @@ Astro 的设计比其他UI框架和语言更简单，其中一个重要原因是
 
 **Astro 是集多功能于一体的Web框架，提供了构建网站所需的一切** Astro 包括组件语法、基于文件的路由、资产处理、构建处理、捆绑、优化、数据获取等。您可以在不超过 Astro核心功能集 的情况下构建出色的网站。
```
Comment (zihan-ch) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:77]:
```suggestion
**Astro 是集多功能于一体的 Web 框架，提供了构建网站所需的一切** Astro 包括组件语法、基于文件的路由、资产处理、构建处理、捆绑、优化、数据获取等。您可以在不超过 Astro 核心功能集 的情况下构建出色的网站。
```

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 3]
```diff
@@ -74,6 +76,6 @@ Astro 的设计比其他UI框架和语言更简单，其中一个重要原因是
...
 
-如果您需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), [image optimizations](https://www.npmjs.com/package/@astrojs/images)等[100多个集成](https://astro.build/integrations/) 扩展 Astro 只需要一个命令 [即可连接您喜欢的CMS](https://astro.build/integrations/) 或 [部署到您喜欢的服务器](/zh-cn/guides/deploy/)
+如果您需要更多的控制，你可以通过 [React](https://www.npmjs.com/package/@astrojs/react), [Svelte](https://www.npmjs.com/package/@astrojs/svelte), [Vue](https://www.npmjs.com/package/@astrojs/vue), [Tailwind CSS](https://www.npmjs.com/package/@astrojs/tailwind), [MDX](https://www.npmjs.com/package/@astrojs/mdx), [image optimizations](https://www.npmjs.com/package/@astrojs/image)等[100多个集成](https://astro.build/integrations/) 扩展 Astro 只需要一个命令 [即可连接您喜欢的CMS](https://astro.build/integrations/) 或 [部署到您喜欢的服务器](/zh-cn/guides/deploy/)
 
 Astro 与 UI 无关，这意味着您可以自带UI框架（BYOF）。React、Preact、Solid 、Svelte、Vue和 Lit都在Astro中得到官方支持。您甚至可以在同一页面上混合和匹配不同的框架，使未来的迁移更容易，并防止项目锁定到单个框架。
```
Comment (zihan-ch) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:81]:
```suggestion
Astro 与 UI 无关，这意味着您可以自带 UI 框架（BYOF）。React、Preact、Solid 、Svelte、Vue 和 Lit 都在 Astro 中得到官方支持。您甚至可以在同一页面上混合和匹配不同的框架，使未来的迁移更容易，并防止项目锁定到单个框架。
```

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 4]
```diff
@@ -20,8 +20,10 @@ Astro 是**集多功能于一体的Web框架**，用于构建快速、以内容
 
 相比之下，大多数现代 Web框架 都是为构建 Web应用程序 而设计的，这些框架最适合构建更复杂、类似应用程序的体验；管理仪表板、收件箱、社交网络、待办事项列表，甚至是像 [Figma](https://figma.com) 和 [Ping](https://ping.gg/) 这样的本地应用程序。
```
Comment (zihan-ch) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:21]:
```suggestion
相比之下，大多数现代 Web框架 都是为构建 Web应用程序 而设计的，这些框架最适合构建更复杂、类似应用程序的体验；管理仪表板、收件箱、社交网络、待办事项列表，甚至是像 [Figma](https://figma.com) 和 [Ping](https://ping.gg/) 这样的体验接近于原生应用的 Web 应用程序。
```

File: src/content/docs/zh-cn/concepts/why-astro.mdx [hunk 5]
```diff
@@ -61,7 +63,7 @@ Astro 的魔力在于它如何将上述两个值（内容焦点于服务器优
 
 我们首先确保您可以使用您已经知道的任何喜欢的 UI 组件语言。在 Astro 中创建新的 UI 组件时都支持React、Preact、Svelte、Vue、Solid、Lit 和其他一些组件。
```
Comment (zihan-ch) [inline src/content/docs/zh-cn/concepts/why-astro.mdx:64]:
```suggestion
我们首先确保您可以使用您已经知道的任何喜欢的 UI 组件语言。在 Astro 中创建新的 UI 组件时都支持 React、Preact、Svelte、Vue、Solid、Lit 和其他一些组件。
```

**Review by zihan-ch — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
Thanks you review @zihan-ch

**Review by Yan-Thomas — APPROVED** [review]
Nice work @liruifengv, @zihan-ch 💜 Time to merge! 🚀

**Pull request merged**